java-squid/src/main/java/org/sonar/java/ast/parser/TreeFactory.java

```diff
@@ -187,6 +197,10 @@ public ExpressionTree newPackageDeclaration(Optional<List<AnnotationTreeImpl>> a
     return (ExpressionTree) partial;
   }
 
+  public ImportClauseTree newEmptyImport(AstNode semicolonTokenAstNode) {
+    return (ImportClauseTree) new EmptyStatementTreeImpl(semicolonTokenAstNode);
```

Comment:
:new: ![MINOR](https://raw.githubusercontent.com/henryju/image-hosting/master/minor.png)Remove this unnecessary cast to "ImportClauseTree".[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1905)